src/rocker/core.py

```diff
@@ -117,7 +117,9 @@ def check_args_for_activation(cls, cli_args):
         return True if cli_args.get(cls.get_name()) else False
 
     @staticmethod
-    def register_arguments(parser, defaults={}):
+    def register_arguments(parser, defaults:dict=None):
```

Comment:
```suggestion
    def register_arguments(parser, defaults: dict=None):
```

Need a space for readability.